projects/kubernetes/kops/patches/0001-remove-hardcoded-requierment-on-flannel-plugin.patch

```diff
@@ -1,28 +1,26 @@
-From 2ec5afbedada5688c13aa8e8d9ac9ffba3ae4511 Mon Sep 17 00:00:00 2001
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
```

Comment:
Is this allowed? Like a wildcard or something?

Reply:
We do it off the EKS-D Kubernetes patches